src/Controller/BarcodeController.php

```diff
@@ -65,9 +65,11 @@ public function barcodeAction(
         $this->taskManager->scan($task);
         $this->doctrine->getManager()->flush();
 
+        $iri = $iriConverter->getIriFromItem($task);
         return $this->json([
-            "ressource" => $iriConverter->getIriFromItem($task),
+            "ressource" => $iri,
             "client_action" => $clientAction,
+            "token_action" => hash('xxh3', BarcodeUtils::getToken($iri)),
```

Comment:
maybe add a comment to explain what this token is about?

Reply:
https://github.com/coopcycle/coopcycle-web/pull/4766/commits/f31cfdd8339cda315ed11ae7a6926eb240361b11#diff-ce8ab65090e1c1907d56d08bbc3d1d313960dcc803973d848c2a9b9d125ca231R72-R85